pipelines/lib/util.groovy

```diff
@@ -322,6 +322,55 @@ def void runPublish(Map p) {
   )
 } // runPublish
 
+/**
+ * Loads LSSTCAM test data
+ * @param buildDir where to run this
+ * @param testDir where to place the test data
+ * @return full path of test data
+ */
+def loadLSSTCamTestData(
+  String buildDir,
+  String testDir){
+  def gcp_repo = 'ghcr.io/lsst-dm/docker-gcloudcli'
```

Comment:
Seems a bit weird to have `rclone` inside `docker-gcloudcli`.  But I guess it's not part of native distributions.